doc/_static/skymap/showVisitSkyMap.py

```diff
@@ -0,0 +1,683 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+#
+
+
+import argparse
+import matplotlib
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+import numpy as np
+from astropy import units
+from astropy.coordinates import SkyCoord
+from matplotlib.legend import Legend
+
+import lsst.afw.cameraGeom as cameraGeom
+import lsst.daf.butler as dafButler
+import lsst.geom as geom
+import lsst.log as lsstLog
+import lsst.sphgeom as sphgeom
+
+logger = lsstLog.Log.getLogger("showVisitSkyMap.py")
+
+
+def bboxToRaDec(bbox, wcs):
+    """Get the corners of a BBox and convert them to lists of RA and Dec."""
+    sphPoints = wcs.pixelToSky(geom.Box2D(bbox).getCorners())
+    ra = [float(sph.getRa().asDegrees()) for sph in sphPoints]
+    dec = [float(sph.getDec().asDegrees()) for sph in sphPoints]
+    return ra, dec
+
+
+def percent(values, p=0.5):
```

Comment:
"percent" is not a very descriptive name for this function (also, since it's presumably accepting values from 0-1, it's not actually a percentile anyway?). Maybe a more descriptive name like "get_value_at_percentile"?

Reply:
Preexisting, but totally agree!  Updated.